### PR TITLE
Add StoryTeller Engine

### PR DESCRIPTION
**Pull request by rich1richard**

### CODE REVIEW

**Comment by rich1richard** [comment]
Hi,
I don't understand! Does this request have a problem? Or is there a step I missed?

**Comment by Calinou** [comment]
> Hi,
> I don't understand! Does this request have a problem? Or is there a step I missed?

I didn't see the original notification, sorry. I get a lot of notifications in a daily basis :slightly_smiling_face:

File: README.md [hunk 1]
```diff
@@ -175,6 +175,7 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
 - [Scrolling Background](https://github.com/dploeger/godot-scrollingbackground) - A scrolling background node (Godot 2.1,3.x).
 - [GodotNotificationCenter](https://github.com/didier-v/GodotNotificationCenter) - A notification center (Godot 3.x).
 - [Ridiculous Coding](https://github.com/jotson/ridiculous_coding) - Makes your coding experience in Godot 1000× more ridiculous (Godot 3.x).
+- [StoryTeller Engine](https://github.com/HeavenMercy/StoryTeller-Engine) - A Dialog and Interactive Fiction engine (Godot 3.x)
```
Comment (Calinou) [inline README.md]:
```suggestion
- [StoryTeller Engine](https://github.com/HeavenMercy/StoryTeller-Engine) - Dialog and interactive fiction engine (Godot 3.x).
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**